docs/guide/introduction.rst

```diff
@@ -34,7 +34,7 @@ The output of **setup** is an :class:`.AlchemicalNetwork`. This contains all
 the information about what is being simulated (e.g., what ligands, host proteins, solvation details etc) and the
 information about how to perform the simulation (the Protocol).
 
-The output of the **executation** stage is the basic results from each edge.
+The output of the **execution** stage is the basic results from each edge.
```

Comment:
😂 This had to be me writing, but why was I repeatedly typing "executation"? Thanks for the typo fixes!

Reply:
I was like, "is this a word with a slightly different meaning that no-one understands the distinction of", like dynamic vs dynamical or alchemic vs alchemical :laughing: